From: Miaohe Lin <linmiaohe@huawei.com>
Subject: mm/mm_init.c: update obsolete comment in get_pfn_range_for_nid()
Date: Sun, 25 Jun 2023 11:33:40 +0800

Since commit 633c0666b5a5 ("Memoryless nodes: drop one memoryless node boot
warning"), the warning for a node with no available memory is removed.
Update the corresponding comment.

Link: https://lkml.kernel.org/r/20230625033340.1054103-1-linmiaohe@huawei.com
Signed-off-by: Miaohe Lin <linmiaohe@huawei.com>
Cc: Mike Rapoport (IBM) <rppt@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/mm_init.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/mm/mm_init.c~mm-mm_initc-update-obsolete-comment-in-get_pfn_range_for_nid
+++ a/mm/mm_init.c
@@ -1677,8 +1677,7 @@ static inline void alloc_node_mem_map(st
  *
  * It returns the start and end page frame of a node based on information
  * provided by memblock_set_node(). If called for a node
- * with no available memory, a warning is printed and the start and end
- * PFNs will be 0.
+ * with no available memory, the start and end PFNs will be 0.
  */
 void __init get_pfn_range_for_nid(unsigned int nid,
 			unsigned long *start_pfn, unsigned long *end_pfn)
_
